docs/source/raft_configuration.md

```diff
@@ -233,10 +219,9 @@ include the endpoint of the new Raft orderer.
 
 Removing a node from a Raft cluster is done by:
 
-  1. Removing its endpoint from the channel config for all channels, including
-  the system channel controlled by the orderer admins.
+  1. Removing its endpoint from the channel config for all channels.
   2. Removing its entry (identified by its certificates) from the channel
-  configuration for all channels. Again, this includes the system channel.
+  configuration for all channels.
   3. Shut down the node.
 
 Removing a node from a specific channel, but keeping it servicing other channels
```

Comment:
Shouldn't we update this section to also include the `osnadmin channel remove` function?